article/app/topmentions/TopMentionsService.scala

```diff
@@ -28,7 +28,13 @@ class TopMentionsService(topMentionsS3Client: TopMentionsS3Client) extends GuLog
     topMentions.get().flatMap(_.get(blogId))
   }
 
-  def getAllTopMentions(): Option[Map[String, TopMentionsDetails]] = {
+  def getTopicList(blogId: String): Option[Seq[TopicWithCount]] = {
```

Comment:
This is great, do you think it's worth adding a test for it?

Reply:
yes!